great-idea-website/css/index.css

```
@@ -63,4 +63,76 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.container {
+	border: 1px solid red;
+	margin: 0 auto; 
+}
+
```

```suggestion
.container {
	border: 1px solid red;
	margin: 0 auto; 
}
``` here you need a width soo that its not the full page